db/migrations/migrations.go

```diff
@@ -0,0 +1,36 @@
+package migrations
+
+import (
+	"database/sql"
+	"embed"
+	"fmt"
+	"math"
+
+	"github.com/pressly/goose/v3"
+)
+
+//go:embed *.sql
```

Comment:
Does this work? Reason I ask is I've found go embedding to be cumbersome sometimes and needing to embed in the `cmd` directory (which is why I've stored the HTML templates in `cmd/main`

Reply:
It seemed to work when I tried it - it's relative to the `.go` file that has the `//go:embed` comment, so you could actually have the templates in another directory as long as it has a `.go` file with the embed export.

Reply:
```sh
$ dropdb showtime && createdb showtime

$ go run cmd/init/init.go
2022/12/12 13:45:22 OK    00001_initialise.sql
2022/12/12 13:45:22 goose: no migrations to run. current version: 1
2022/12/12 13:45:22 successfully initialised showtime

```

(not sure what the "no migrations to run" line is all about, but it does definitely run them!)

Reply:
Weird, makes it seem like it's got some sort of baseline and it's ignoring? But if it works then no issue!

You might be able to use the standalone goose tool to get a bit more insight into the state

Reply:
It seems to always print it when it's done:

https://github.com/pressly/goose/blob/b5826417d982a065ecbfa3d0d714ef2997a9775d/up.go#L112-L119

Reply:
Ah, makes sense!